samples/WebApi/README.md

```diff
@@ -0,0 +1,136 @@
+# Description
+This is a sample application, which uses CoreRT to compile a .NET Core Web API sample.
+TODO WIP - Might not work on Linix
```

Comment:
Is the TODO still applicable?

Reply:
Will work once the newest CoreFX and CoreRT packages are built and published to MyGet.